tss-esapi-sys/build.rs

```diff
@@ -112,3 +227,81 @@ pub fn generate_from_system(esapi_out: PathBuf) {
         .write_to_file(esapi_out)
         .expect("Couldn't write ESYS bindings!");
 }
+
+#[cfg(all(feature = "generate-bindings", feature = "bundled"))]
+pub fn generate_from_system(esapi_out: PathBuf) {
+    let mut clang_args: Vec<String> = Vec::new();
+    #[cfg(windows)]
+    {
+        let hklm = winreg::RegKey::predef(winreg::enums::HKEY_LOCAL_MACHINE);
+        let ip = hklm.open_subkey("SOFTWARE\\WOW6432Node\\Microsoft\\Microsoft SDKs\\Windows\\v10.0").unwrap();
+        let ip: String = ip.get_value("InstallationFolder").unwrap();
+        let ip_pb = PathBuf::from(ip).join("Include");
+        let windows_sdk = ip_pb.join("10.0.17134.0");
+        clang_args.push(format!("-I{}", windows_sdk.join("ucrt").display()));
+        clang_args.push(format!("-I{}", windows_sdk.join("um").display()));
+        clang_args.push(format!("-I{}", windows_sdk.join("shared").display()));
+    }
+
+    let out_path = std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap());
+    let p = out_path.join("tpm2-tss");
+    let tss2_esys_include_path = p.join("include").into_os_string().into_string().unwrap();
```

Comment:
This I do not understand. They all seem to be the same.

Reply:
Yeah I forgot to simplify those.